tests/integration/testdata/sync/code/after/makefile_layer_create_new_file/Makefile

```diff
@@ -0,0 +1,7 @@
+build-HelloWorldLayer:
```

Comment:
does this work on windows too?

Reply:
good question, have not addressed this one yet because I'm not able to access my Windows VM at the moment

Reply:
done; addressed by changing `touch` to instead echo into a file (should be supported in windows too)